pyproject.toml

```diff
@@ -28,7 +28,7 @@ include = [
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 # Used for local development
-nautobot = "^2.1.2"
+nautobot = "^2.0.6"
```

Comment:
Why this change? Why 2.0.6 in particular?

Reply:
Reverted the change made by dependabot. `2.0.6` was there before as it was the latest `2.0` version without security issues AFAIK.

Reply:
The reason for the revert was that there are more changes to be made to support `2.1.2`, and I didn't want to mix it here. Should I update it in a separate PR, or leave it for the release? WDYT @glennmatthews